File: backend/booking-service/src/main/java/com/tripify/booking_service/dto/CartItemDTO.java
package com.tripify.booking_service.dto;

import java.math.BigDecimal;
import java.time.LocalDate;

public record CartItemDTO(
        Long id,
        Long catalogItemId,
        Integer quantity,
        BigDecimal priceAtAdded,
        Long roomTypeId,
        Long fareClassId,
        LocalDate checkIn,
        LocalDate checkOut
) {}

File: frontend/app/src/main/java/com/tripify/tripify_android/data/model/CartItemDTO.kt
package com.tripify.tripify_android.data.model

data class CartItemDTO(
    val id: Long,
    val catalogItemId: Long,
    val quantity: Int,
    val priceAtAdded: Double,

    val currency: String? = null,
    val roomTypeId: Long? = null,
    val fareClassId: Long? = null,
    val checkIn: String? = null,
    val checkOut: String? = null
)
